apps/src/templates/teacherDashboard/urlHelpers.js

```diff
@@ -5,7 +5,7 @@ const dashboardPrefix = '/teacher_dashboard/sections/';
  */
 export const teacherDashboardUrl = (sectionId, path = '') => {
   // Prepend a forward slash to path if one is not supplied.
-  if (path && path.at(0) !== '/') {
```

Comment:
fun fact: the original code here does not appear to be valid javascript and I have no idea how this worked previously either in our tests or on our site. this code is executed on the teacher homepage, so its not an obscure codepath.

Reply:
I did notice the existence of one implementation of `String.prototype.at`: https://github.com/mathiasbynens/String.prototype.at its possible babel 6 allowed this, but I didn't investigate to confirm this since it didn't seem necessary.